src/main/java/org/prebid/server/handler/ExceptionHandler.java

```diff
@@ -24,11 +25,18 @@ public static ExceptionHandler create(Metrics metrics) {
 
     @Override
     public void handle(Throwable exception) {
-        logger.warn("Generic error handler: {0}, cause: {1}",
-                errorMessageFrom(exception), errorMessageFrom(exception.getCause()));
+        if (!isConnectionResetException(exception)) {
```

Comment:
Let's consider creation of `shouldLogException()` and inside it check if it `isConnectionResetException`.
Possible that in the future we will have more than one exception that we should not log